pkg/env/core.go

```diff
@@ -49,13 +49,7 @@ func (c *core) GetCertificateSecret(ctx context.Context, secretName string) (*rs
 
 func NewCore(ctx context.Context, log *logrus.Entry) (Core, error) {
 	deploymentMode := deployment.NewMode()
```

Comment:
there's another one of these in cmd/aro/operator.go which you might as well get too